grails-app/conf/Config.groovy

```diff
@@ -1091,6 +1091,18 @@ openboxes.browser.connection.status.timeout = 8000
 //google.analytics.enabled = false
 //google.analytics.webPropertyID = "UA-xxxxxx-x"
 
+// Default webhook configuration
+openboxes {
+    webhook {
+        enabled = false
+        endpoint {
+            url = 'https://webhook.site/9969a2d6-9780-483d-beb4-50667d66497d'
```

Comment:
Do you want to have these values here? Or it should be empty string as a default?

Reply:
I was ok with leaving it as an example since the endpoint is disabled by default.